packages/components/pages/f-checkout/src/components/Address.vue

```diff
@@ -54,6 +55,12 @@ export default {
             fieldType: 'address',
             fieldLength: '50'
         };
+    },
+
+    methods: {
+        addFocus () {
+            document.getElementById('formField-line-1').focus();
```

Comment:
I think this method should either be in the checkout-form-field or we should expose the name of the field and reference it by that as we have a bit of ikky coupling as it is.